src/main/java/org/folio/circulation/domain/notice/TemplateContextUtil.java

```diff
@@ -108,6 +109,29 @@ public static JsonObject createStaffSlipContext(Request request) {
     return createStaffSlipContext(request.getItem(), request);
   }
 
+  public static JsonObject addPrimaryServicePointNameToStaffSlipContext(JsonObject entries, ServicePoint primaryServicePoint) {
+    log.debug("addPrimaryServicePointNameToStaffSlipContext:: parameters entries: {} primaryServicePoint: {}", entries, primaryServicePoint);
+    if (primaryServicePoint == null) {
+      log.info("addPrimaryServicePointNameToStaffSlipContext:: primaryServicePoint object is null");
+      return entries;
+    }
+
+    if (entries == null) {
+      log.info("addPrimaryServicePointNameToStaffSlipContext:: entries JsonObject is null, primaryServicePointName: {}", primaryServicePoint.getName());
+      return new JsonObject();
+    }
+
+    entries.getJsonArray(PICK_SLIPS_KEY)
+      .stream()
+      .map(JsonObject.class::cast)
+      .map(pickSlip -> pickSlip.getJsonObject(ITEM))
+      .forEach(item -> item.put("effectiveLocationPrimaryServicePointName", primaryServicePoint.getName()));
+
+    log.debug("addPrimaryServicePointNameToStaffSlipContext:: Result after adding Primary Service Point Name to Staff Slip Json:: entries: {}, primaryServicePointName: {}", entries, primaryServicePoint.getName());
```

Comment:
please change log level to info and additionally, consider simplifying the log message to"addPrimaryServicePointNameToStaffSlipContext:: result entries: {}, primaryServicePointName: {}", entries, primaryServicePoint.getName()